manifests/mod/dav_svn.pp

```diff
@@ -21,7 +21,7 @@
 
   ::apache::mod { 'dav_svn': }
 
-  if $::osfamily == 'Debian' and ! ($::operatingsystemmajrelease in ['6', '9', '16.04', '18.04']) {
+  if $::osfamily == 'Debian' and ! ($::operatingsystemmajrelease in ['6', '9', '10', '16.04', '18.04']) {
```

Comment:
This feels like a workaround on some older Debian versions while the newer ones behave as you would expect. Perhaps it should be checked which versions need this. I'd expect Ubuntu 20.04 behaves the same as Debian 10.

Then you would get to something like:
```suggestion
  if $::osfamily == 'Debian' and $::operatingsystemmajrelease in ['7', '8', '14.04'] {
```

Untested, but I think you get the idea.

Reply:
Yeah I got it. I also did found some patches from Debian for the dav_svn module which should fix this in the package itself, but the issue still reproduces on my Debian 10 testing machine.

Reply:
I just checked. The Debian 10 package has pretty similar word-around in mods-available/dav_svn.load, but that file is overwritten by this puppet module.

Reply:
I must admit I haven't looked at the dav_svn module for ages. Perhaps you can write an acceptance test for the module so we know it really works rather than guessing? That also gives the best chance against regressions.

Reply:
I will give a shot, It's not hard to reproduce since apache2 simply won't start when including apache::mod::dav_svn :D

Reply:
Exactly, those are the things that are fairly easy to test.

Reply:
Given the result of the acceptance test and looking at how Debian implemented their work-around it's safe to say that the whole version checking part of that if statement can be removed. Agreed?

Reply:
I think that makes sense but it's probably good to check with the actual maintainers.

Reply:
Thanks for the fix @martijndegouw and the assistance during the PR @ekohl . The test is great - allows us to check we haven't inadvertently stomp over some logic put in there for the older versions, as you said @ekohl .